sycl/test/xocc_tests/sdaccel_ports/vision/edge_detection/edge_detection_with_webcam.cpp

```diff
@@ -0,0 +1,108 @@
+#include <algorithm>
+#include <cstdlib>
+#include <fstream>
+#include <iostream>
+#include <iterator>
+#include <string>
+
+#include <CL/sycl.hpp>
+#include "device_selectors.hpp"
+
+// OpenCV Includes
+#include <opencv2/opencv.hpp>
+#include <opencv2/core/core.hpp>
+#include <opencv2/imgproc/imgproc.hpp>
+#include <opencv2/highgui/highgui.hpp>
+
+using namespace cl::sycl;
+
+class krnl_sobel;
+
+int main(int argc, char* argv[]) {
+
+  constexpr auto height = 480; // input.rows;
+  constexpr auto width = 640; // input.cols;
+  constexpr auto area = height * width;
+
+  cv::VideoCapture cap;
+  // set buffer to 1 frame
+  cap.set(cv::CAP_PROP_BUFFERSIZE, 1);
+  if(!cap.open(0))
+    return 0;
+
+  // set the size of picture taken by the webcam
+  cap.set(cv::CAP_PROP_FRAME_WIDTH, width);
+  cap.set(cv::CAP_PROP_FRAME_HEIGHT, height);
+
+  selector_defines::CompiledForDeviceSelector selector;
+  queue q {selector, property::queue::enable_profiling()};
```

Comment:
```suggestion
  queue q { selector, property::queue::enable_profiling {} };
```
to look more like modern C++, to make more obvious `property::queue::enable_profiling` is a type